web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -2815,8 +2814,6 @@ private void updateQuery(GenericResponse<String> response, RunningQuery runningQ
                             maxResultsOverride, parameters);
             
             // Fire off an audit prior to updating
-            Set<String> methodAuths = new HashSet<>(Arrays.asList(q.getQueryAuthorizations().split("\\s*,\\s*")));
-            cbAuths.retainAll(methodAuths);
```

Comment:
Bug was...
```
[ERROR] java.util.HashSet<java.lang.String> is incompatible with expected argument type java.util.HashSet<java.util.Collection<java.lang.String>> .... at QueryExecutorBean.java:[line 2819] GC_UNRELATED_TYPES
```
...but I then discovered the real bug was that `cbAuths` is never used again in this method. So, this update necessitated the change to ExtendedQueryExecutorBeanTest.java